packages/fx-core/src/question/other.ts

```diff
@@ -1066,3 +1066,48 @@ function oauthClientSecretQuestion(): TextInputQuestion {
     },
   };
 }
+
+export function syncManifestQuestionNode(): IQTreeNode {
+  return {
+    data: {
+      type: "group",
+    },
+    children: [
+      {
+        data: {
+          type: "folder",
+          name: QuestionNames.ProjectPath,
+          title: getLocalizedString("core.syncManifest.projectPath"),
+          cliDescription: "Project Path",
+          placeholder: "./",
+          default: "./",
```

Comment:
should the default be the same as default folder when TTK creates a new project?

Reply:
Thanks for pointing out. Will change this.